src/http.c

```diff
@@ -92,13 +92,14 @@ int http_send(struct tunnel *tunnel, const char *request, ...)
 		url_encode(password, tunnel->config->password);
 		pwstart = strstr(logbuffer, password);
 
-		if (pwstart != NULL) {
+		while (pwstart != NULL) {
```

Comment:
Perhaps change:
```c
		pwstart = strstr(logbuffer, password);

		while (pwstart != NULL) {
			...
			pwstart = strstr(logbuffer, password);
		}
```
to:
```c
		while (pwstart = strstr(logbuffer, password)) {
			...
		}
```

Reply:
I had something similar before and ran into compiler warnings. `gcc` suggests to put parentheses around the assignment used as truth value. With this change it is happy now :-)